formats/conf.py

```diff
@@ -44,7 +44,7 @@
     # API
     'javadoc' : (downloads_root + '/latest/bio-formats5.1/api/%s', ''),
     # Doc links
-    'omero_doc' : (oo_site_root + '/support/omero5.1/%s', ''),
+    'omero_doc' : (oo_site_root + '/support/omero5.2/%s', ''),
```

Comment:
I wonder whether it would actually be better to remove the version numbers here altogether - links only go to the model, units & structured annotations sections of the dev docs anyway.